packages/components/src/components/carousel/carousel.tsx

```diff
@@ -122,13 +127,13 @@ export class Carousel {
       perMove: 1,
       mediaQuery: 'min',
       padding: {
-        right: '7%', // together with wrapContent this is overridden via css
+        right: spacingFluidLarge, // together with wrapContent this is overridden via css
       },
       // TODO: this uses matchMedia internally, since we also use it, there is some redundancy
-      // TODO: for gap definition the gridGap const must be used
+      // TODO: for gap definition the gridGap const must be used - will be done after Grid Refactoring
       breakpoints: getSplideBreakpoints(this.slidesPerPage as Exclude<BreakpointCustomizable<number>, string>, {
-        base: spacingStaticSmall,
-        s: spacingStaticMedium,
+        base: spacingStaticMedium,
+        s: spacingStaticLarge,
```

Comment:
Breakpoint `l` is now redundant with `s` and can be removed if it is following a mobile first approach which seems to be the case based on `mediaQuery: 'min'` option.

Reply:
Fixed it